autoware_utils/include/autoware_utils/geometry/alt_geometry.hpp

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020-2024 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef AUTOWARE_UTILS__GEOMETRY__ALT_GEOMETRY_HPP_
+#define AUTOWARE_UTILS__GEOMETRY__ALT_GEOMETRY_HPP_
+
+#include "autoware_utils/geometry/boost_geometry.hpp"
+
+#include <list>
+#include <optional>
+#include <utility>
+#include <vector>
+
+namespace autoware_utils
```

Comment:
The namespace for all the header files needs to be `autoware::utils`

```suggestion
namespace autoware::utils
```